### PR TITLE
add PUT to update disaster attributes and fix disasterMap json

### PR DESCRIPTION
**Pull request by kradical**

### CODE REVIEW

File: meteor/models.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def to_dict(self):
             'created_at': self.created_at,
             'updated_at': self.updated_at,
             'disaster_type': self.disaster_type,
+            'cluster_activated': self.cluster_activated,
```
Comment (kradical) [inline meteor/models.py]: oops, somehow missed this first time.

File: meteor/urls.py [hunk 2]
```diff
@@ -13,11 +13,11 @@
...
-    url(r'^users/([0-9]+)/$', views.user_id, name="user_id"),
+    url(r'^users/([0-9]+)$', views.user_id, name="user_id"),
 
     url(r'^organizations$', views.organization, name="organization"),
-    url(r'^organizations/([0-9]+)/$', views.organization_id, name="organization_id"),
+    url(r'^organizations/([0-9]+)$', views.organization_id, name="organization_id"),
 
     url(r'^disasters$', views.disaster, name="disaster"),
-    url(r'^disasters/([0-9]+)/$', views.disaster_id, name="disaster_id")
+    url(r'^disasters/([0-9]+)$', views.disaster_id, name="disaster_id")
```
Comment (kradical) [inline meteor/urls.py]: took the trailing / off these urls, it was causing issues.

File: meteor/views.py [hunk 3]
```diff
@@ -127,12 +127,15 @@ def get_user(request):
...
-        return JsonResponse({'error': 'NO_PERMISSION'}, status=401)
     
     try:
         user = User.objects.get(id=user_id)
+
+        if user == request.user:
+            return JsonResponse(user.to_dict())
+
+        if not request.user.permission == User.ADMIN:
+            return JsonResponse({'error': 'NO_PERMISSION'}, status=401)
```
Comment (kradical) [inline meteor/views.py]: A user can get their own user info by id

File: meteor/views.py [hunk 4]
```diff
@@ -222,6 +224,7 @@ def create_earthquake(data, user):
             created_by=user,
             magnitude=magnitude,
             epicenter=geocoords)
+        new.disaster_type = DisasterMap.EARTHQUAKE
```
Comment (kradical) [inline meteor/views.py]: another big oops.

**Comment by kradical** [comment]
Overall I'm starting to see some code duplication so may consider refactoring, but I think splitting the views into a couple files soon is also a priority.

File: meteor/models.py (hunk 1 above)
Comment (ProgrammingJoe) [inline meteor/models.py]: We might not be using it for awhile anyway

File: meteor/views.py [hunk 5]
```diff
@@ -262,19 +265,124 @@ def get_disasters(request):
...
+    try:
+        disaster = DisasterMap.objects.get(id=disaster_id)
+
+        if disaster.disaster_type == DisasterMap.EARTHQUAKE:
+            disaster = EarthquakeMap.objects.get(id=disaster_id)
+
+        if request.user != disaster.created_by and not request.user.permission in (User.TRUSTED, User.ADMIN):
+            return JsonResponse({'error': 'NO_PERMISSION'}, status=401)
+
+        disaster.delete()
```
Comment (ProgrammingJoe) [inline meteor/views.py]: Does this cascade from the earthquake to the general disaster map?

Comment (kradical) [inline meteor/views.py]: We discussed this in slack, summary: no it doesn't.

**Pull request merged**